p2p/server.go

```diff
@@ -809,10 +836,8 @@ func (srv *Server) SetupConn(fd net.Conn, flags connFlag, dialDest *discover.Nod
 		return
 	}
 
-	// if sql database handle is available, update node information
-	if srv.DB != nil {
```

Comment:
I don't get why you remove the check here, but you check `srv.DB != nil` earlier in this method.

Reply:
Similar to other comments above, the checking is now done inside the `srv.storeNodeInfo` function where sql update functions are called.
Basically, all db handle checking should now occur where sql update functions, like `addNodeMetaInfo`, `addNodeInfo`, `updateNodeInfo`, are directly called.
The `srv.DB` is checked earlier in this method because of `srv.addNodeMetaInfo`.